### DISCUSSION TITLE
Future of the projects

### DISCUSSION THREAD

**Discussion by lucasgrjn** [facebookresearch/Khronos.jl#17] [Q&A]
Hi @smartalecH

Thanks for sharing Khronos! It's really cool to see a new FDTD solver opened (especially for GPUs/multi-plaforms).
I just wanted to know what the direction of this one will be. Can we expect to see new features released soon like multi-GPU versions? Or is this the final code released as it is?

Anyway, thanks again for all of that!
Regards,
Lucas

**Comment by smartalecH** [comment]
Hi @lucasgrjn, thank you for the kind words!

Yes, we plan to continue maintaining and adding functionality. In particular, we want to focus on multi-gpu parallelization by hiding the communication cost behind the computation and building out an adjoint module for inverse design capabilities. There are a few other important features, like more boundary conditions and lorentzian susceptibilities, which we want to deploy too.

It's always hard to provide timelines for OSS projects like this because company priorities are fluid and constantly changing. But we think it's important to grow the computational photonics community, and this is one way to do so. 

Note that there are a lot of other "low hanging fruit" community contributions (like a Documenter.jl setup) which could help us gain more internal traction. It's easier for us to allocate more resources to a project if we can demonstrate engagement thats benefiting our other responsibilities.

**Reply by jjBoca** [reply]
Hello, are there any further updates you can share at this time, either regarding scope or timeline? Specifically, is there any intent to add nonlinear susceptibilities/ polarization options  or even carrier injection / semiconductor dynamics going forward? Thank you!

**Comment by lucasgrjn** [comment]
Hey @smartalecH 

Thanks for the response, I see you already have a nice roadmap defined!

I know how it is complicated internally, so I simply prefered to ensure it was actively on development. (Coupling Khronos.jl with [Layout.jl](https://www.youtube.com/live/v0RPD4eSzVE?si=BHcFqT4IxzNNH6FP&t=8085), if released, could be really interesting to see.)

For the "low hanging fruit" part, I am really busy now but I see and definitely agree with your point. More examples from the community could also be possible and interesting.

(Thanks to move this issue to the discussion tab to avoid the flood!)
